lte/gateway/c/core/oai/tasks/amf/amf_app_state_manager.cpp

```diff
@@ -110,7 +101,9 @@ int AmfNasStateManager::initialize_state(const amf_config_t* amf_config_p) {
 
   // Allocate the local AMF state and create respective single object
   create_state();
-
+#if MME_UNIT_TEST
```

Comment:
Is the intent to build this during the unit testing and not during the normal build? Besides this, I also see that the flag is used inside `read_state_from_db()` already, why is it checked here again?

Reply:
Yes, the intent is to build this only during the unit testing and not during the normal build. Since this acts as a base for all the upcoming changes(`read_ue_state_from_db `does not have` #if !MME_UNIT_TEST `check at state manager class), this check acts as a double-check to make sure stateless changes do not affect the amf application code. This check is removed in the Stateless Integration PR and only the check inside `read_state_from_db()` will stay.

Reply:
Sure, added the required comment.